src/main/java/core/mate/academy/model/Truck.java

```diff
@@ -1,15 +1,29 @@
 package core.mate.academy.model;
 
+import java.util.List;
+
 /**
  * Add some custom fields that could be only in Truck
  * Do not remove no-field constructor
  */
 public class Truck extends Machine {
+    private int wheelsQuantity;
+    private double cargoVolume;
+
     public Truck() {
     }
 
     @Override
     public void doWork() {
         System.out.println("Truck started to work");
     }
+
+    @Override
+    public List<Machine> get() {
```

Comment:
The `get` method is not typically a method you would override in a class like `Truck`. This method seems to be creating instances of `Truck` and returning them as a list, which is not a standard behavior for a class representing a single machine. Consider removing this method or ensuring it aligns with the intended design pattern.